src/modules/auth/home/HomeScreen.js

```diff
@@ -312,29 +312,33 @@ function HomeScreen(props) {
 								</ListGroupItem>
 
 								<ListGroupItem className="mb-0">
-									<Link to="/manage-totp" className="d-block">
-										{t('HomeScreen|Manage OTP')}
-										<CustomInput
-											className="float-right"
-											type="switch"
-											id="otpSwitch"
-											name="otpSwitch"
-											defaultChecked={userinfo?.available_factors && userinfo.available_factors.indexOf("totp") != -1 ? true : false}
-										/>
-									</Link>
+									<Row className="align-items-center">
+										<Col sm={6}>
+											<Link to="/manage-totp" className="d-block">
+												{t('HomeScreen|Manage OTP')}
+											</Link>
+										</Col>
+										<Col sm={6}>
+											<div className="float-right text-muted pointer" id="webauthn" name="webauthn">
+												<div className={((userinfo?.available_factors) && (userinfo.available_factors.indexOf("totp") != -1)) ? `status-circle status-green` : `status-circle`} title={((userinfo?.available_factors) && (userinfo.available_factors.indexOf("totp") != -1)) ? t("HomeScreen|Active") : t("HomeScreen|Inactive")}/>
```

Comment:
@Aringoaway Can you split that to more rows for better legibility? The line is too long